### ISSUE TITLE
Having issues with learning rate normalization

### ISSUE THREAD

**Issue by guilhermesilveira** [autonomio/talos#60]
Thanks so much for coming here to raise an issue. Please take a moment to 'check' the below boxes:

- [X] I'm up-to-date with the latest release:
- [X] I've confirmed that my Keras model works outside of Talos.

-------------------------------------------------------------------------

https://gist.github.com/guilhermesilveira/5e2f3ae4f60a2929897f8ef4da99a73e

The above example shows a sample usage of `lr_normalizer` which seems to not be working as expected (I might be wrong, but the issue might be with the comparison itself). THe code should return 1 (1000/1000) but returns 1000.

**Comment by mikkokotila** [comment]
Can you try:

```
import keras
import talos 

talos.model.lr_normalizer(1, keras.optimizers.Adam)
```
It should return 0.001 which is the default (1x) learning rate for Adam.

**Comment by guilhermesilveira** [comment]
It works, returning 0.001. But:

```
import talos
import tensorflow.keras as keras
talos.models.lr_normalizer(1, keras.optimizers.Adam)
```

Returns 1

```
import talos
import  keras
talos.models.lr_normalizer(1, keras.optimizers.Adam)
```

Returns 0.001. It is somehow related to the weird `import` I've used on the first example.
